ci-operator/jobs/infra-periodics.yaml

```diff
@@ -2734,70 +2734,11 @@ periodics:
   labels:
     ci-operator.openshift.io/cluster: multi01
     ci.openshift.io/role: infra
-  name: periodic-trigger-ci-tools-build-root-mabc
+  name: periodic-trigger-images-mabc
```

Comment:
Definitely you'll prove me wrong here but `periodic-trigger-mabc-images` sounds more correct in English.